tests/test_templating.py

```diff
@@ -114,6 +114,18 @@ def test_escaping_without_template_filename(app, client, req_ctx):
     assert flask.render_template("mail.txt", foo="<test>") == "<test> Mail"
 
 
+def test_escaping_svg(app, client):
```

Comment:
Don't need this test, it's just testing that Jinja works. At most, you wouuld test that `app.select_jinja_autoescape` returned `True` for a `.svg` filename.